scripts/get_latest_dockerhub_version.py

```diff
@@ -22,4 +26,12 @@
         numeric_tags.append(tag_to_store)
 
 numeric_tags.sort(key=lambda s: list(map(int, s.split('.'))), reverse=True)
-print(numeric_tags[0])
+
+if check_version == "latest":
+    print(numeric_tags[0]) # print latest version
+else:
+    # prints true if the version is in the list but not latest
+    # printing true means publish.sh sync a non-latest version
+    is_latest = (numeric_tags[0] == check_version)
+    is_published = check_version in numeric_tags
+    print(str(not is_latest and is_published).lower()) # print if version is latest
```

Comment:
Maybe remove this comment or specify:
- print true if version is not latest and already published

Reply:
sure that's more clear

Reply:
I've also now added more description in code comment at the beginning of the script